vocab/credentials/v2/vocabulary.yml

```
@@ -23,6 +23,16 @@ class:
     deprecated: true
     comment: A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
 
+  - id: JsonSchema2023
+    label: JSON schema validator 2023
+    upper_value: cred:CredentialSchema
+    comment: A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
+
+  - id: CredentialSchema2023
```

```suggestion
  - id: VerifiableCredentialSchema2023
```

Same suggestion as mentioned here: https://github.com/w3c/vc-data-model/pull/1178/files#r1257273645

---

I'm ok with this but need to change it in the spec itself first